essence-of-live-coding/src/LiveCoding/Cell/Profunctor.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE DeriveAnyClass #-}
+
+-- export only instances
```

Comment:
Ok, but we should export them always. I think it's best to avoid the orphan instance and add this instance to `Cell.lhs`.

Reply:
I mean, you've added it to `LiveCoding.hs`, which is great for users, but it also means that we can't import it internally in the library (because that would be a circular import).

Reply:
So I should move all the code to `Cell.lhs`, correct ?

Reply:
Yes.